### PR TITLE
docs: cleanup in the PDE section related to PPD-913

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR cleans up old content related to PDE that is no longer needed or caused confusion. 
💻 [Preview](https://deploy-preview-1145--docs-spectrocloud.netlify.app/devx/enterprise-user)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1145#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by tiwarisumit3 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1145#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1145#pullrequestreview-1308267873" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx.md
Comment (ritawatson) [inline content/docs/04.5-devx.md:39]:
```suggestion
Refer to [PDE Use Cases](/devx/enterprise-user) to learn more about different use cases of PDE.
```

File: content/docs/04.5-devx/02-enterprise-user.md
Comment (ritawatson) [inline content/docs/04.5-devx/02-enterprise-user.md:19]:
```suggestion
You can use the Palette Developer Experience (PDE) to serve two common use cases: enterprise developers and individual application authors who want to deploy a containerized application without worrying about infrastructure overhead. 
```

File: content/docs/04.5-devx/02-enterprise-user.md
Comment (ritawatson) [inline content/docs/04.5-devx/02-enterprise-user.md:50]:
```suggestion
Use PDE to deploy your containerized applications to Palette. Leverage Palette's free-tier offering of PDE to get started. Create your application profiles and deploy your applications to Palette in no time.
```

**Review by ritawatson — APPROVED** [review]
Looks great with just a couple comments.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1145#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1145#pullrequestreview-1308271013" title="Approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1145#pullrequestreview-1308267873" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
